app/views/layouts/application.html.erb

```diff
@@ -105,8 +105,30 @@
   </div>
 </div>
 <% unless internal_navigation? %>
+  <script>
```

Comment:
This code is not fully correct and I am unable to find a better solution. 
The problem is mentioned here:

https://www.loom.com/share/c4e77d518beb42ba93ad2fae3116241e?sid=2189e46b-b40e-4f1c-9ff4-bea56dd04436